src/backend/question/serializers.py

```diff
@@ -36,11 +43,70 @@ def get_question_des(self, obj):
     def get_category_name(self, obj):
         return obj.category.name
 
+    def get_suggestion_level(self, solutionObj, userObj, paramObj={
```

Comment:
`SolutionSerializer` should only use for data serialization, we can implement the business login using `action` (https://www.django-rest-framework.org/api-guide/viewsets/) in `SolutionViewSet`